examples/glusterfs/README.md

```diff
@@ -2,15 +2,45 @@
 
 [Glusterfs](http://www.gluster.org) is an open source scale-out filesystem. These examples provide information about how to allow containers use Glusterfs volumes.
 
-The example assumes that the Glusterfs client package is installed on all nodes.
+The example assumes that you have already set up a Glusterfs server cluster and the Glusterfs client package is installed on all Kubernetes nodes.
 
 ### Prerequisites
 
-Install Glusterfs client package on the Kubernetes hosts.
+Set up Glusterfs server cluster; install Glusterfs client package on the Kubernetes nodes. ([Guide](https://www.howtoforge.com/high-availability-storage-with-glusterfs-3.2.x-on-debian-wheezy-automatic-file-replication-mirror-across-two-storage-servers))
+
+### Create endpoints
+Here is a snippet of glusterfs-endpoints.json,
+```
+      "addresses": [
+        {
+          "IP": "10.240.106.152"
+        }
+      ],
+      "ports": [
+        {
+          "port": 1,
+          "protocol": "TCP"
+        }
+      ]
+
+```
+The "IP" field should be filled with the address of a node in the Glusterfs server cluster. The "port" field is not really used, so any value between 1 and 65535 will work.
```

Comment:
Then, the statement that "port field is not really used" is not true.
I will suggest changing it to say something like: "In this example, it is fine to give any valid value (from 1 to 65535) to the port field"